substrate/frame/revive/rpc/src/client.rs

```diff
@@ -324,7 +336,7 @@ impl Client {
 		let client = self.clone();
 		spawn_handle.spawn("subscribe-blocks", None, async move {
 			let res = client
-				.subscribe_new_blocks(|block| async {
+				.subscribe_new_blocks(SubscriptionType::BestBlocks, |block| async {
```

Comment:
Why are we not using finalized here?

Reply:
that for the "in memory" cache maintained by the eth-rpc. In Ethereum you don't wait for the block to be finalized when you call the RPC, so I am not waiting for finalized blocks here. We could decide to wait for the finalized block but that would also impact the user's experience, you will see MM spinning for much longer.

Reply:
Okay no its fine then. It wasn't apparent from the function names.